### ISSUE TITLE
balanced correlations

### ISSUE THREAD

**Issue by juandavidgutier** [ronikobrosly/causal-curve#44]
Hello @ronikobrosly 

I am new with Generalized Propensity Score (GPS), and I am reproducing the script of the example named NHANES_BLL_example.ipynb, of the module causal-curve. My question is how I can get the adjusted (balanced) correlations between the treatment (Blood lead levels) and the potential confounders in the example (Age, Sex_Male, Race_NH_Black, Race_NH_White, Race_Other, Edu_HS, Edu_LT_HS, Smoke_Home_Yes, Baby_NICU_Yes, Food_Often_bad, Food_Sometimes_bad), or how can I evaluate the performance of covariates balance?

I will appreciate a lot your cooperation.

Regards,

**Comment by ronikobrosly** [comment]
Hi @juandavidgutier , thanks for reaching out. This isn't something I've implemented yet, but this is a very good idea. I did a little googling and came across this paper: https://arxiv.org/pdf/1812.06575.pdf
It seems to discuss approaches to assessing covariate balance. I'm really busy these days but can eventually get to it. If you discover an approach and want to contribute here I would love the help!
